lib/Sema/ConstraintSystem.h

```diff
@@ -407,8 +407,10 @@ enum ScoreKind {
   SK_ScalarPointerConversion,
   /// A conversion from an array to a pointer of matching element type.
   SK_ArrayPointerConversion,
+  /// A conversion to an empty existential type ('Any' or '{}').
```

Comment:
Oh this is a partial revert of code that I pulled out a while back. I take that to mean empty protocol composition.